__tests__/app.test.js

```diff
@@ -96,6 +96,26 @@ describe("/api/articles/10000000000", () => {
   });
 });
 
- 
+
+describe('/api/articles', () => {
+  test('GET - status 200 - responds with an array of arcticle objects', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(result => {
+        expect(result.body.articles.length).toBe(12);
+              result.body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.comment_count).toBe("number");
```

Comment:
I would also add a test to check this specific property is correct - for instance checking [0] in the return data and checking the comment_count property is as expected

Reply:
This would need to be [0] once sorted